### PR TITLE
Collapse block style and feature update

### PR DESCRIPTION
**Pull request by codewisenate**
- Collapse block updated to add BC Gov styling option
- Added ability to toggle the first item open as an option
- Deprecated version added to provide better experience for existing content
- Includes Menu Manager isset check for editor roles

### CODE REVIEW

**Comment by codewisenate** [comment]
image

**Comment by codewisenate** [comment]
image

File: src/Actions/MenuEditor.php [hunk 1]
```diff
@@ -200,7 +200,7 @@ public function add_editor_menu_capabilities() {
 
         $current_user = wp_get_current_user();
 
-        if ( current_user_can( 'editor' ) && ( $current_user->caps['editor'] ) ) {
+        if ( current_user_can( 'editor' ) && ( isset( $current_user->caps['editor'] ) ) ) {
```
Comment (mhaswell-bcgov) [inline src/Actions/MenuEditor.php:203]:
```suggestion
        if ( current_user_can( 'editor' ) && ( $current_user->has_cap( 'editor' ) ) ) {
```
Are we able to use the WP_User->has_cap() (https://developer.wordpress.org/reference/classes/WP_User/has_cap/) function here?

**Review by mhaswell-bcgov — APPROVED** [review]

File: src/Actions/MenuEditor.php (hunk 1 above)
Comment (codewisenate) [reply src/Actions/MenuEditor.php:203]: This one was giving me some problems locally (story of my life). I wouldn't mind digging into it a bit more, but for now checking the capabilities index seems to do the trick.

**Pull request merged**